src/explore-education-statistics-frontend/src/hooks/useCookies.ts

```diff
@@ -42,6 +44,20 @@ export const allowedCookies: AllowedCookies = {
       expires: addYears(new Date(), 10),
     },
   },
+  userTestingBannerSeen: {
+    name: 'ees_user_testing_banner_seen',
+    duration: '1 year',
+    options: {
+      expires: addYears(new Date(), 1),
+    },
+  },
+  disableUserTestingBanner: {
+    name: 'ees_disable_user_testing_banner',
+    duration: '1 year',
+    options: {
+      expires: addYears(new Date(), 1),
+    },
+  },
```

Comment:
What is this for? We don't use it and we don't set it anywhere so it might have been left in by mistake?